packages/gatsby-plugin-typescript/package.json

```diff
@@ -10,6 +10,7 @@
     "Noah Lange <noahrlange@gmail.com>"
   ],
   "dependencies": {
+    "@babel/plugin-proposal-object-rest-spread": "^7.0.0",
     "@babel/preset-typescript": "^7.0.0",
     "@babel/runtime": "^7.0.0",
     "babel-plugin-remove-graphql-queries": "^2.0.2-rc.3"
```

Comment:
This plugin doesn't seem to use this package anymore. Is it safe to remove?

Reply:
Should be safe - this plugins should be added by gatsby core : https://github.com/gatsbyjs/gatsby/blob/dfeea37ac3e64b40752fb399d8c9d161ded71169/packages/gatsby/src/utils/babel-loader-helpers.js#L26-L31